### PR TITLE
Fixed issue with Dialog field validation passing when no values selected

### PR DESCRIPTION
**Pull request by chalettu**
@miq-bot add_label bug
BZ https://bugzilla.redhat.com/show_bug.cgi?id=1512398

### CODE REVIEW

**Comment by martinpovolny** [comment]
@chalettu : please, rebase. Whom do you suggest for review? Thn!

**Comment by chalettu** [comment]
@eclarizio Do you think you can take a quick look at this PR and review?

File: src/dialog-user/services/dialogData.ts [hunk 1]
```diff
@@ -112,7 +112,7 @@ export default class DialogDataService {
       if (field.type === 'DialogFieldCheckBox' && fieldValue === 'f') {
         validation.isValid = false;
         validation.message = __('This field is required');
-      } else if (fieldValue === '') {
+      } else if (_.isEmpty(fieldValue)) {
```
Comment (ohadlevy) [inline src/dialog-user/services/dialogData.ts]: @chalettu mind adding a test? thanks

Comment (chalettu) [reply src/dialog-user/services/dialogData.ts]: @ohadlevy , just added a test for this scenario

**Comment by martinpovolny** [comment]
Adding Gaprindashvili/yes, please, ping me if that is wrong.

**Comment by himdel** [comment]
@miq-bot add_label gaprindashvili/backported
@miq-bot remove_label gaprindashvili/yes

This was released in manageiq-ui-components 1.0.2, backported to gaprindashvili via https://github.com/ManageIQ/manageiq-ui-classic/pull/2951.

**Pull request merged**